Core/CoreTests/Inbox/ComposeMessage/ComposeMessageViewModelTests.swift

```diff
@@ -641,6 +641,15 @@ class ComposeMessageViewModelTests: CoreTestCase {
         // Then
         XCTAssertTrue(testee.searchedRecipients.isEmpty)
     }
+
+    func test_selectedRecipients_disableToggle() {
+        // Given
+        let recipient = ReceiptStub.getRecipientExceedMaxLimit()
+        // When
+        testee.selectedRecipients.send(recipient)
+        // Then
+        XCTAssertTrue(testee.isSendIndividualToggleDisabled)
```

Comment:
1. I think we should also test whether `sendIndividual` is set to true in this case.
2. And maybe that it's set back to the previous value after limit is no longer exceeded.